content/docs/w5yi/images/vec_w5yi.svg

```diff
@@ -0,0 +1,454 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
```

Comment:
```suggestion
	 viewBox="0 0 128 128" width="50px" style="enable-background:new 0 0 128 128;" xml:space="preserve">
```
Maybe something like setting the [width](https://stackoverflow.com/questions/3120739/resizing-svg-in-html)

Reply:
I tested the above locally and 250px width looks kinda nice. You can run `hugo server` locally to preview. See instructions in `maintainer.md`

Reply:
I have update the .svg and index.md file.    It would not pull the .svg initially because it kept putting w5yi instead of W5YI in the path.   It not checks out with hugo!!